

include/asm/unistd.h: In function `execve':
include/asm/unistd.h:441: `__ERRNO_LAST' undeclared (first use in this function)
include/asm/unistd.h:441: (Each undeclared identifier is reported only once
include/asm/unistd.h:441: for each function it appears in.)

I'm too wimpy to do the nested include.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-generic/errno.h |    2 --
 25-akpm/include/asm-i386/unistd.h   |    2 +-
 2 files changed, 1 insertion(+), 3 deletions(-)

diff -puN include/asm-i386/unistd.h~make-key-management-use-syscalls-not-prctls-build-fix include/asm-i386/unistd.h
--- 25/include/asm-i386/unistd.h~make-key-management-use-syscalls-not-prctls-build-fix	2004-09-06 23:51:38.789986384 -0700
+++ 25-akpm/include/asm-i386/unistd.h	2004-09-06 23:53:19.576664480 -0700
@@ -309,7 +309,7 @@
 
 #define __syscall_return(type, res) \
 do { \
-	if ((unsigned long)(res) >= (unsigned long)(-(__ERRNO_LAST + 1))) { \
+	if ((unsigned long)(res) >= (unsigned long)(-(128 + 1))) { \
 		errno = -(res); \
 		res = -1; \
 	} \
diff -puN include/asm-generic/errno.h~make-key-management-use-syscalls-not-prctls-build-fix include/asm-generic/errno.h
--- 25/include/asm-generic/errno.h~make-key-management-use-syscalls-not-prctls-build-fix	2004-09-06 23:53:14.356458072 -0700
+++ 25-akpm/include/asm-generic/errno.h	2004-09-06 23:53:18.408842016 -0700
@@ -101,6 +101,4 @@
 #define	EKEYREVOKED	127	/* Key has been revoked */
 #define	EKEYREJECTED	128	/* Key was rejected by service */
 
-#define	__ERRNO_LAST	128
-
 #endif
_
